GermanFilter/sections/specific.txt

```diff
@@ -67,6 +67,7 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
 !
 carsharing-news.de##.inside-right-sidebar > aside#custom_html-2 > .textwidget > a[href][target="_blank"] > img
 coolis.de##.td-ss-main-sidebar > aside.widget_text > .textwidget > .g-1 > .g-single > a.gofollow[href] > img
+stuttgarter-zeitung.de##.mod-teaser--sondt
```

Comment:
Also not shown.

Reply:
<details>
<summary>Here it is</summary>
<img src="https://user-images.githubusercontent.com/18536328/73923539-15903a00-48dc-11ea-8c1c-606b6d6feb84.png"/>
</details>